src/js/controller/captions.js

```diff
@@ -17,11 +17,12 @@ define([
         // Listen for provider subtitle tracks
         //   ignoring provider "subtitlesTrackChanged" since index should be managed here
         _model.mediaController.on('subtitlesTracks', function(e) {
+            _model.mediaController.off('meta');
```

Comment:
It would be good to add the function here in case other modules are listening for the meta event:

`_model.mediaController.off('meta', _metaHandler);`